src/components/posts-list/PostsList.js

```diff
@@ -4,46 +4,71 @@ import PostCard from '../post-card/PostCard';
 
 class PostsList extends Component {
   state = {
-    posts: []
-    // todo 2: добавить isLoading индикатор
+    posts: [],
+    // ttodo 2: добавить isLoading индикатор
+    isLoading: false
   };
 
   componentDidMount() {
-    // todo 2: вызвать загрузку постов
+    // ttodo 2: вызвать загрузку постов
+      this.loadPosts()
   }
 
   loadPosts = async () => {
-    // todo 2:
+    // ttodo 2:
     //  проверьте лежит ли ваш accessToken в constants/index.js
+      !accessToken && console.warn('No Posts...');
     //  прежде чем отправить запрос - включите в true флажок загрузки в стейте isLoading
+      this.setState({isLoading:true});
     //  выполните fetch запрос за поcтами на `https://gorest.co.in/public-api/posts?access-token=${accessToken}`
+      fetch(`https://gorest.co.in/public-api/posts?access-token=${accessToken}`)
+          .then((response) => {
+              return response.json()
+          })
+          .then((data) => {
+              console.log('data',data);
+              this.setState({
+                  posts:data.result || [],
+                  isLoading: false
+              })
+          })
     //  похожий запрос выполнялся в компоненте PostCard в функции loadComments
     //  результат выполнения запроса нужно положить в стейт в posts
     //  когда запрос выполнится - не забудьте поменять индикатор загрузки isLoading на false
   };
 
+  loading = () => {
+      return (
+          <div className="text-center">
+              <span className='text-primary'>Loading...</span>
+              <div className="spinner-border text-primary" role="status">
+                  <span className="sr-only">Loading...</span>
+              </div>
+          </div>
+      )};
```

Comment:
the same as in previous file

when you have same code in different files in most cases it's a sign that it should be refactored and reused